week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,47 @@
-// Write your code here
+
+
+let slideIndex = 1;
+let timer = null;
+
+
+function plusSlides(n) {
+  clearTimeout(timer);
+  showSlides(slideIndex += n);
+}
+
+function currentSlide(n) {
+  clearTimeout(timer);
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  let i;
+  let slides = document.getElementsByClassName("mySlides");
+  let dots = document.getElementsByClassName("dot");
+
+  if (n==undefined){
```

Comment:
```suggestion
  if (n == undefined){
```
Careful with the spacing here